src/main/java/org/alfasoftware/soapstone/SoapstoneOpenApiReader.java

```diff
@@ -413,6 +423,40 @@ private RequestBody parametersToRequestBody(String operationId, Collection<Param
     JavaType javaType = soapstoneConfiguration.getObjectMapper().constructType(type);
     LOG.debug("          " + javaType.toString());
 
+    XmlJavaTypeAdapter adapterAnnotation = currentResourceClass.getPackage().getAnnotation(XmlJavaTypeAdapter.class);
+    if (adapterAnnotation == null || adapterAnnotation.type() != type) {
+      XmlJavaTypeAdapters adaptersAnnotation = currentResourceClass.getPackage().getAnnotation(XmlJavaTypeAdapters.class);
+
+      final Type finalType = type;
+      adapterAnnotation = adaptersAnnotation == null ? null : Arrays.stream(adaptersAnnotation.value())
+          .filter(adAnn -> adAnn.type() == finalType)
+          .findFirst().orElse(null);
+
+      Converter<?, ?> memberConverter = null;
```

Comment:
`memberConverter` doesn't make sense in this context as we are not dealing with a member in the same way we are in `ParentAwareModelResolver`.

Reply:
have renamed to parameterTypeConverter. Does that make sense?